#!/usr/bin/python3
def raise_exception():
    r = 5 + "hello"
